### ISSUE TITLE
Train Cargo Station / Cargo Harbor incoming/outgoing

### ISSUE THREAD

**Issue by CityGecko** [chronofanz/EnhancedDistrictServices#55]
Would it be possible to assign Warehouses to/from cargo train station/cargo harbor as incoming/outgoing ?

**Comment by dakotahawkins** [comment]
Yeah I really need this. I don't think it sees my cargo train stations as valid places to take product, even if it's inside the same district.
